test/nn/simplicial/test_dist2cycle.py

```diff
@@ -0,0 +1,44 @@
+"""Unit tests for Dist2Cycke Model."""
+import numpy as np
+import numpy.linalg as npla
+import torch
+from toponetx.classes import SimplicialComplex as sc
+
+from topomodelx.nn.simplicial.dist2cycle import Dist2Cycle
+
+
+class TestDist2Cycle:
+    """Unit tests for the Dist2Cycle model class."""
+
+    def test_forward(self):
+        """Test the forward method of Dist2Cycle."""
+        edge_set = [[1, 2], [1, 3], [2, 5], [3, 5]]
+        face_set = [[2, 3, 4], [2, 4, 5]]
+
+        torch.manual_seed(42)
+        ex2_sc = sc(edge_set + face_set)
+        ld = ex2_sc.down_laplacian_matrix(rank=1).todense()
```

Comment:
`ld` is not the convention used in TopoModelX to refer to the laplacian.
See for example:
https://github.com/pyt-team/TopoModelX/blob/486adf2299eb37aa22ec1fdf54351458074da253/topomodelx/nn/simplicial/sccnn_layer.py#L266

In your case here, use instead: `laplacian_down_1.

Note: In Python, it is best practice to avoid short variable names. Because: (1) they are not informative for users / readers of the code and make everything confusing. (2) "Find and replace" on short variable names would change a lot of things in the code. Not convenient.